src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -15,7 +17,8 @@ namespace NzbDrone.Core.MediaFiles.EpisodeImport
 {
     public interface IMakeImportDecision
     {
-        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, QualityModel quality = null);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, ParsedEpisodeInfo folderInfo);
```

Comment:
`ParsedEpisodeInfo` `folderInfo` I notice you use folder info in other places too.

maybe a better name for the class would be `ParsedPathInfo`?